packages/client/src/monaco-language-client.ts

```diff
@@ -77,41 +66,8 @@ export class MonacoLanguageClient extends BaseLanguageClient {
         });
     }
 
-    protected doCreateConnection(): Thenable<IConnection> {
-        const errorHandler = (this as any).handleConnectionError.bind(this);
-        const closeHandler = this.handleConnectionClosed.bind(this);
-        return this.connectionProvider.get(errorHandler, closeHandler, this.outputChannel);
-    }
-
     protected createMessageTransports(encoding: string): Promise<MessageTransports> {
-        throw new Error('Unsupported');
-    }
-
-    protected registerBuiltinFeatures(): void {
-        super.registerBuiltinFeatures();
```

Comment:
Does the base implementation now perform this registration and it is therefore no longer needed here?

Reply:
Ok, I looked at `vscode-languageclient\lib\common\client.js` and this is done there. Was that an error to register it twicebefore or was that really an change in the lc lib?

Reply:
I don't really know, it seems there isn't any consequence of registering a feature twice, so I don't know for how long it was there.

I think these features were `proposed` back then, so we added them and forgot to remove it when they became builtin